tools/autodeployment/server.js

```diff
@@ -5,9 +5,14 @@ const createHandler = require('github-webhook-handler');
 const shell = require('shelljs');
 const mergeStream = require('merge-stream');
 const fs = require('fs');
+const { firstCheck } = require('./src/sibling-checker');
 
 const { buildStart, buildStop, handleStatus } = require('./info');
 
+// Server health check
+
+firstCheck();
```

Comment:
You could just call it `healthCheck()`

Reply:
Sure. It will make more sense in this scope.